docs/integrations/databricks.md

```diff
@@ -80,7 +80,7 @@ the Unity Catalog metadata.
 
 lakeFS support for Unity Catalog differs between lakeFS OSS and lakeFS Enterprise & Cloud.
 
-### Catalog Exports <span class="badge">Open-Source</span>
+### Catalog Exports <span class="badge">lakeFS</span>
```

Comment:
I think this is `lakeFS Core` now? @ozkatz

Reply:
Decided on lakeFS